python/pyspark/sql/tests/connect/streaming/test_parity_foreach_batch.py

```diff
@@ -136,6 +138,29 @@ def func(df, _):
             if q:
                 q.stop()
 
+    def test_streaming_foreach_batch_external_column(self):
```

Comment:
Could you move this to `pyspark/sql/tests/streaming/test_streaming_foreach_batch.py`? The test will be shared between classic and connect.

Reply:
We can remove this now as the super method will be called automatically.